odxtools/admindata.py

```diff
@@ -74,6 +81,15 @@ def from_et(et_element: ElementTree.Element, doc_frags: List[OdxDocFragment]) ->
 
         return Modification(change=change, reason=reason)
 
+    def _build_odxlinks(self) -> Dict[OdxLinkId, Any]:
+        return {}
+
+    def _resolve_odxlinks(self, odxlinks: OdxLinkDatabase):
+        pass
+
+    def _resolve_snrefs(self, diag_layer: "DiagLayer"):
+        pass
```

Comment:
Any reason why would this class have references in the future?

Reply:
Even _build_odxlinks is emtpy

Reply:
The idea is that this looks like a "simple class", that don't need to participate in the resolution logic